README.rst

```diff
@@ -5,6 +5,12 @@
 
 |Latest Documentation Status| |Stable Documentation Status| |Gitter chat| |CI| |Build status|
 
+|Repology| |PyPI package| |Conda-forge| |Python Versions|
```

Comment:
A bit too late, but shouldn't we first put our official PyPI package, rather than all the things we don't control?